index.html

```diff
@@ -332,23 +437,193 @@ <h4>SvgRenderingTemplate</h4>
-  }]
+  <span class="highlight">"renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    "template": {
+      "id": "https://example.edu/credentials/BachelorDegree.svg",
+      "mediaType": "image/svg",
+      "digestMultibase": "zQmerWC85Wg6wFl9znFCwYxApG270iEu5h6JqWAPdhyxz2dR",
+      "renderProperty": [
+        "/issuer", "/validFrom", "/credentialSubject/degree/name"
+      ]
+    }
+  }
   </span>
 }
         </pre>
 
         <p>
-In the example above, the <a>issuer</a> has provided an SVG rendering template
-for a bachelor's degree that will be filled in with the information in the
-<a>verifiable credential</a>. The rendering is preferred when the device or
-paper is in a portrait (display is taller than it is wider) orientation.
+In the example above, the [=issuer=] has provided a Mustache-based SVG rendering
+template for a Bachelor's degree that will be filled in with specific
+information from the [=verifiable credential=] listed in `renderProperty`.
         </p>
+
+        <section>
+          <h3>The `svg-mustache` Render Suite</h3>
+          <p>
+The `svg-mustache` render suite uses the Mustache templating language to
+modify an SVG file, which is then used to render a visual representation
+of the [=verifiable credential=].
+          </p>
+
+          <p>
+In the example below, a fully embedded SVG file is used as the rendering
+template.
+          </p>
+
+          <pre class="example nohighlight"
+               title="Basic usage of the svg-mustache render suite">
+{
+  <span class="comment">...</span>
+  "renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    <span class="comment">// the SVG file is embedded in the VC</span>
+    "template": "data:image/svg;base64,Qjei89...3jZpW"
+  }
+}
+          </pre>
+
+          <p>
+The next example links to the SVG file on the Web and secures it against
+modification by using the `digestMultibase` property.
+          </p>
+
+          <pre class="example nohighlight"
+              title="A remotely hosted SVG file for an SVG render template">
+{
+<span class="comment">...</span>
+"renderMethod": {
+  "type": "TemplateRenderMethod",
+  "renderSuite": "svg-mustache",
+  "template": {
+    <span class="comment">// this SVG file is fetched from the Web</span>
+    "id": "https://degree.example/bachelors.svg",
+    "mediaType": "image/svg",
+    "digestMultibase": "zQmerWC85Wg6wFl9znFCwYxApG270iEu5h6JqWAPdhyxz2dR"
+  }
+}
+          </pre>
+
+          <p>
+The next example links to the rendering template on the Web and secures it
+using the `digestMultibase` property:
+          </p>
+
+          <pre class="example nohighlight"
+              title="A remotely hosted SVG render method">
+{
+<span class="comment">...</span>
+"renderMethod": {
+  <span class="comment">// this render method is fetched from the Web</span>
+  "id": "https://degrees.example/bachelors-svg.jsonld",
```

Comment:
```suggestion
  "id": "https://degrees.example/bachelors-svg.jsonld",
  "mediaType": "application/ld+json",
```

Maybe we also drop `.jsonld` here...but it might be helpful to call out more quickly that this is _not_ an SVG file.